docs/sections/user_guide/cli/tools/fs/copy-exec-no-target-dir-err.cmd

```diff
@@ -1 +1 @@
-uw fs copy --config-file copy-config.yaml config files
+uw fs copy --config-file copy-config.yaml --key-path config.files
```

Comment:
Here's an example of the updated CLI usage, replacing positional `KEY` arguments with an explicit `--key-path` argument, identical to that used in `config` mode and by the various drivers.